src/endpoints/metrics.js

```diff
@@ -40,8 +40,8 @@ const getResourceQuery = (resourceType) => {
  */
 const getDateQuery = (startDate, endDate) => {
   return {
-    $gt: new Date(startDate),
-    $lt: new Date(endDate),
+    $gte: new Date(startDate),
+    $lte: new Date(endDate),
   }
```

Comment:
see https://docs.mongodb.com/manual/reference/operator/query/lte/ and https://docs.mongodb.com/manual/reference/operator/query/gte/